### PR TITLE
#163259823 setup database 

### PR DESCRIPTION
**Pull request by wathigo**
### What does this PR do?
Sets up the database for the application.

### Description of the task to be completed.
1. Creation of a database Questioner.
2. Setting up configurations for the database.
3. Testing the database using user sign up endpoint.

### Any background context?
None

### How can this be manually tested?
Open the terminal.
Enter git clone https://github.com/wathigo/Questioner.git
Create a .env file in the app directory.
Enter 'heroku config -s >> .env' to load the enviroment variables.
Enter 'flask run'
Test the endpoints using postman.

### Relevant PT stories
#163259823

### Screenshots
screenshot from 2019-01-17 01-14-06
screenshot from 2019-01-17 01-14-17

### Questions:
What do you think about this implementation?

### CODE REVIEW

**Comment by colMike** [comment]
You have set up the db effectively. Good work

**Comment by MainaEK** [comment]
Nice work

File: app/api/v1/models/models_user.py [hunk 1]
```diff
@@ -1,28 +1,32 @@
...
 
-    def authenticate_user(self, email, password):
-        """ Check if user exists and compare password"""
+        save = self.db
+        cur = save.cursor()
+        cur.execute(query)
+        save.commit()
+        return data
+
+    """def authenticate_user(self, email, password):
```
Comment (ronaldndirangu) [inline app/api/v1/models/models_user.py]: Why have you commented this code out?

**Review by ronaldndirangu — CHANGES_REQUESTED** [review]

**Comment by ronaldndirangu** [comment]
Ensure you fix the travis build failing

File: app/api/v1/models/models_user.py (hunk 1 above)
Comment (wathigo) [reply app/api/v1/models/models_user.py]: I have not modified the post-login endpoint to use a database. I did that in order to test for user registration only to ensure that my connection to the database is working.

**Pull request merged**